src/styles/_color.scss

```
@@ -41,6 +41,9 @@ $red-10: #a51111;
 $orange-soft: #ffece2;
 $orange-hard: #f38119;
 
+// Yellow
+$yellow: #ffd43b;
+
```

```suggestion
// enterprise
$kakao: #ffd43b;

```
혹시 변수 이름 이렇게 바꿔주실 수 있나요? 저희가 현재 사용하는 컬러는 기본 노랑이라기보다는 카카오 노랑이라고 생각해서요 😊 
추후 `facebook`, `google` 등이 추가될 때 각 enterprise별로 컬러 코드를 저장하기에도 편리합니다! 🎉 
선언된 위치도 색상이 모여있는 곳 말고 다른 위치로 옮겨주시면 감사하겠습니다 🙇

---

넵